src/pages/namsunkim/Feed/Components/Comment/Comment.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import './Comment.scss';
+
+export default class Comment extends Component {
+
+  render() {
+    const { commentArr, likeComment, deleteComment } = this.props;
+    const { id, isLiked, content, userName } = commentArr;
+
+    return (
+      <>
+        <li>
+          <a href="#!" className="bold black">{userName}</a>
+          <span>{content}</span>
+          <i className="xi-close-min deleteComment pointer"
+            onClick={() => deleteComment(id)}></i>
+          <i className={'xi-heart' + (!isLiked ? '-o' : ' likeComment') + ' likeHeart pointer'}
```

Comment:
```suggestion
          <i className={'xi-heart' + (isLiked ? ' likeComment' : '-o') + ' likeHeart pointer'}
```
사소한 부분이긴 한데 ! 없이 사용할 수 있는 부분같아보이는데 이부분은 !안쓰고 쓰는게 읽고 이해하기에 더 좋아보입니다.

Reply:
넵😊